doc/base/classes.xml

```diff
@@ -20,7 +20,14 @@
 			<argument index="3" name="a8" type="int">
 			</argument>
 			<description>
-				Make a color from red, green, blue and alpha. Arguments can range from 0 to 255.
+				Returns a color made up of red, green, blue and alpha components. Arguments can range from 0 to 255.
+				[b]r8[/b] red component of color
```

Comment:
`red channel` or `red` component should be ok. The convention is to use `'argument_name'` in the docs